tests/testthat/test_createYaml.R

```diff
@@ -0,0 +1,39 @@
+library(PTXQC)
+context("createYaml.R")
+
+test_that("createYaml", {
+
+  ##
+  ##test empty yc object, no parameter or metric specifications
+  ##
+  yc <- YAMLClass$new(list())
+  expect_equal(length(createYaml(yc)$param), 19)
+
+  ##
+  ##test invalid parameter input
+  ##
+  parameter <- list()
+  parameter$nonsense <- c(1,4)
+  expect_equivalent(createYaml(yc, param = parameter)$param$nonsense, NULL)
+  
+  ##
+  ##test valid parameter input
+  ##
+  yc <- YAMLClass$new(list())
+  parameter$param_PG_intThresh <- 30
+  parameter$param_OutputFormats <- "txt"
+  expect_equivalent(createYaml(yc, param = parameter)$param$param_PG_intThresh, 30)

```

Comment:
test the default value for that same parameter as well, and make sure it's not accidentally identical to 30